backend/node/genvm/base.py

```diff
@@ -136,16 +136,17 @@ async def run_contract(
         is_init: bool = False,
         leader_results: None | dict[int, bytes],
         config: str,
-        date: datetime.datetime,
+        date: datetime.datetime | None,
     ) -> ExecutionResult:
         assert date.tzinfo is not None
         message = {
             "is_init": is_init,
             "contract_account": contract_address.as_b64,
             "sender_account": from_address.as_b64,
             "value": None,
-            "datetime": date.isoformat(),
         }
+        if date is not None:
```

Comment:
genvm has a magic default date, so here I just won't place any date if it wasn't given